llama-index-integrations/llms/llama-index-llms-groq/pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.llms.groq"
+
+[tool.llamahub.class_authors]
+groq = "llama-index"
```

Comment:
feel free to add your Github user name here :)

Reply:
Also this should match the class name, which I believe is `Groq`

Reply:
Updated!  Thanks for the feedback.